tests/src/Controller/Client/EditControllerTest.php

```diff
@@ -48,6 +48,9 @@ class EditControllerTest extends TestCase
      */
     protected function setUp(): void
     {
+        // The REQUEST_URI is required either to create the session or to get the selfUrl
+        $_SERVER['REQUEST_URI'] = '/';
```

Comment:
I think this is not necessary any more.

Reply:
@cicnavi if i remove the configuration, although the tests pass I get the following: